### DISCUSSION TITLE
Module Nomiden Not Found

### DISCUSSION THREAD

**Discussion by ammaralwandi97** [teamalgoritma/community#816] [Q&A] [answered]
Minta tolong solusinya, packages nomiden tidak bisa dipakai padahal sudah terinstall
![image](https://github.com/teamalgoritma/community/assets/89758039/1e51084e-2a8c-4da5-82d0-7259d9c7c52f)

**Comment by kinantytasyao** [comment, marked as answer]
Dear @ammaralwandi97 

Hal tersebut karena package nomiden terintall di "base" bukan di environment "dss nik", ketika masuk terminal cmd silakan ikuti langkah-langkah berikut untuk menginstall package nomiden di environment dss_nik
- `conda activate` : jika belum muncul (base) di paling kiri cmd ketika `conda activate`
- `conda activate dss_nik` : mengaktifkan environment dss_nik
- `pip install nomiden` : install package nomiden

Demikian yang dapat kami sampaikan semoga membantu. Jika ternyata jawaban ini menjawab permasalahan Anda silakan mark as answer. Terima kasih
